pyproject.toml

```diff
@@ -34,6 +34,13 @@ documentation = "https://github.io/MeteoSwiss-APN/icon-data-processing-incubator
 # Format: <command> = "<package>.<module>:<function>"
 # icon-data-processing-incubator = "idpi.cli:main"
 
+[tool.setuptools.packages.find]
+namespaces = true
+where = ["src"]
+
+[tool.setuptools.package-data]
+"idpi.data" = ["*.yml"]
```

Comment:
https://setuptools.pypa.io/en/latest/userguide/datafiles.html#subdirectory-for-data-files